### ISSUE TITLE
Fan0 issues with spider board V1.0

### ISSUE THREAD

**Issue by avongil** [FYSETC/FYSETC-SPIDER#70]
My Fan0 mosfet has blown up.  The fan works fine, there are no shorts to it.  The board still works, but there was a small fire.  Interesting that "sleepingAwake" has noise in the Fan0 area.  I am starting to think that this is a flaw in the board.  I am however running 24v.  Moving the hot end fan to the RGB pins. 

> So I too am having fan issues with my spider board purchased through the AliExpress fysetc store. First the boot loader it came with somehow didn’t allow usb or uart communication. Flashing the boot loader they have here got usb and uart working. Anyway the fans. Fan0 is connected to my hot end fan and jumpered to 12v. My print config is copied from here as well with some minor changes but the pins are correct from what I can tell. At first Fan1 wasn’t working. It’s my parts fan. 24v Changing printer.cfg to below worked. I’ll tweak as needed. [fan1] Pb1
> 
> But Fan0 nothing. I’ve tried various config changes and no voltage output.
> 
> Also I’ve noticed when I apply some pressure around the fan0 connector I can hear a faint electric pop. It does it with the fans unplugged to. Could my board have an internal short?

Fan0 may not be coming on due to normal behavior. Shame on me for not noticing the extruder fan behavior below 50c
But my board makes an electrical popping noise if I apply a small amount of pressure around the FAN0 connector and its mosfet. Sounds like the layers are shorting to each other. Its consistent and repeatable. Nothing is loose.

_Originally posted by @sleepingAwake in https://github.com/FYSETC/FYSETC-SPIDER/issues/17#issuecomment-959221344_

**Comment by avongil** [comment]
Here is a picture of the burnt up section of the board.![V1.0 Fire](https://imgur.com/b8WV5Tu.png)

I am assuming the first mosfet if for Fan0 then 1 and 2.  Where are the three other mosfets for the R G and B pins?

**Comment by avongil** [comment]
The svg states that no more than .5 amps should pass through this mosfet.  I will fuse every fan from now on, but I am not so sure the fan was drawing more than .5 amps when this happened.  Is there a possibility that it revived less than 24v?  When this happened the machine shut off after very fast printing.  When I switched it back on I heard a buzzing noise then a poof and flame from the area.    I let the machine cool down for an hour, scraped off the mosfet, then powered on again.   Everything worked normally except for Fan0.

**Comment by avongil** [comment]
Update:  Fan1 (PB0), Fan2(PB1) and Fan3 (PB2)  are dead. Moved them over to the RGB to the right and my fans are once again operational albeit more fussy about pwm.  The rest of the board seems to operate fully.   Not going to rely on it though.  Fysetc sent me a $10 coupon and I upgraded to the Spider V2.2 

I believe this is a board defect around that area since the board does not smell of IC burning, but board burning.
